deployment/packagebuild/packages.d/vnet/openvnet.spec

```diff
@@ -95,6 +113,8 @@ Requires: openvnet-ruby
 # runtime package but we will need it for our 0mq client to work. Why this symlink is included
 # in the development package instead of the runtime package is beyond me.
 Requires: zeromq3-devel
+# for yum-builddep
+BuildRequires: zeromq3-devel
```

Comment:
Wouldn't it be better to keep all BuildRequired together at the top of the file?